src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -255,15 +255,11 @@
         justify-content: space-between;
         margin-top: 10px;
 
-        .share-toggle-container {
+        .toggle-container {
             flex: 0 1 80%;
```

Comment:
these flex styles still work as expected when moving to `toggle-container` right?

Reply:
yup!